RHDEVDOCS-7312: Resolve abstract errors

Dhruv-Soni11 opened the pull request:
This is not a feature request-related PR. It contains fixes for adding the missing [role="_abstract"] tag in all the files.

<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OCPBUGS#<jira-issue-id>][OSDOCS#<jira-issue-id>]: <short-description-of-the-pr> --->

<!--- If your changes apply to the latest release and/or in-development version of OpenShift, open your PR against the `main` branch.
Do not create or rename a top-level directory (or any subdirectory in a directory that contains a hugebook.flag file) in the repository and topic map without checking with a docs program manager first.
If a book is being created or modified, there are changes on the Customer Portal that must also be made.

* For more details about the information requested in this template, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/create_or_edit_content.adoc#submit-PR --->

Version(s):
<!--- Specify the version or versions of OpenShift your PR applies to. -->

gitops-docs-1.20

Issue:
<!--- Add a link to the Bugzilla, Jira, or GitHub issue, if applicable. --->

https://issues.redhat.com/browse/RHDEVDOCS-7312#

Link to docs preview:
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

This pull request does not include any preview files, as it only adds the abstract tag to modules where it was previously missing.

Peer review:
- [ ] Peer has approved this change.
<!--- Peer approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
<!--- Optional: Include additional context or expand the description here.--->

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.--->

Dhruv-Soni11 committed:
Adding the role tag for each file

shivanisathe25 reviewed (commented):
@Dhruv-Soni11 The PR looks good to me. Just wanted to confirm if you want to remove the following files since they are deleted: `.vale.ini ` and `gitops-error.txt`. could you please confirm ?

Dhruv-Soni11 committed:
Merge upstream/gitops-docs-main into resolve-abstract-errors

Resolved conflicts by accepting deletions of 41 unused files that were
removed in PR #107144 (RHDEVDOCS-6565). These files included:

- Old release notes (versions 1.1 through 1.16)
- Deprecated OAuth/Keycloak configuration files
- Other unused documentation modules

Files removed:
- making-open-source-more-inclusive.adoc
- con_routing-traffic-by-using-argo-rollouts.adoc
- gitops-argo-cd-command-line.adoc
- gitops-additional-steps-for-disconnected-clusters.adoc
- gitops-in-built-permissions.adoc
- All release notes from 1.1-1.16 (50+ files)
- OAuth and Keycloak-related files (6 files)

The role tag additions remain for all files that still exist in the
current documentation set.

Resolves conflicts from PR #107144.

Dhruv-Soni11 committed:
Remove gitops-errors.txt file that was accidentally included

The gitops-errors.txt file is a validation report/log file that was
mistakenly included in the original commit. It should not be part of
the repository.

This fixes the CI validation failure.

Dhruv-Soni11 committed:
Fix assembly file to remove references to deleted release notes

The gitops-release-notes.adoc assembly file was referencing release
notes modules (versions 1.11-1.13) that were deleted in PR #107144.
This was causing CI validation failures:

  ERROR: include file not found: modules/gitops-release-notes-1-13-1.adoc
  ERROR: include file not found: modules/gitops-release-notes-1-13-0.adoc
  ... (and 14 more similar errors)

Updated the assembly to only include the release notes that actually
exist in the repository:
- 1.19.0 (latest)
- 1.18.1
- 1.18.0
- 1.17.1
- 1.17.0
- 1.10.3 (legacy)

This completes the cleanup that was started in PR #107144 but missed
updating this assembly file.

Fixes CI validation error.

Dhruv-Soni11 commented:
I am closing this commit as I cannot squash the commits in this PR. It cannot be resolved, and I will create a new PR to fix these errors.
